cmake/partition_manager.cmake

```diff
@@ -208,6 +208,33 @@ fix this, or enable `CONFIG_PM_EXTERNAL_FLASH_SUPPORT_LEGACY` in Kconfig.")
     )
 endif()
 
+# If simultaneous updates of the network core and application core is supported
```

Comment:
Nothing wrong on this line, but this is not sufficient as commit message:
![image](https://user-images.githubusercontent.com/9802583/131987621-5cc4a998-7908-43e9-9eb5-e6c96db972b4.png)

Reply:
My bad, fixed